lib/getTeam.ts

```diff
@@ -0,0 +1,105 @@
+import { Team } from "@prisma/client";
+import { GetServerSidePropsContext } from "next";
+import prisma from "./prisma";
+import logger from "./logger";
+
+const log = logger.getChildLogger({ prefix: ["[lib] getTeam"] });
+
+export const getTeam = async (context: GetServerSidePropsContext): Promise<Team | null> => {
+  let teamIdOrSlug = null;
+  let customDomain = null;
+
+  let team = null;
+
+  log.debug(`{env} ${process.env.NODE_ENV}`);
+
+  switch (process.env.NODE_ENV) {
+    case "development": {
+      if (
+        process.env?.TESTING_TEAMS_CUSTOM_DOMAINS &&
+        Boolean(parseInt(process.env.TESTING_TEAMS_CUSTOM_DOMAINS))
+      ) {
+        teamIdOrSlug = process.env.TESTING_TEAMS_TEAM ?? "1";
+      }
+      break;
+    }
+
+    case "production":
```

Comment:
I wonder if we should hard code calendso.com in here

Reply:
So there's a couple ways to do this. Either adding "domains" as a environment variable or/and adding this code only to the fork we're creating.